civihr_default_theme/scss/partials/_button.scss

```diff
@@ -0,0 +1,16 @@
+.btn-default {
+  border-color: $gray;
+  color: $gray;
+
+  &.active,
+  &.active:hover {
+    border-color: $gray;
+    background: $gray;
+    color: $light;
+  }
+}
+
+.btn-primary{
+  border-color: transparent;
```

Comment:
I think this should be a variable value like `$btn-primary-color` (although it's likely that we want the border color to be transparent for every contextual variation of a button - success, info, etc -)